repositories.json

```diff
@@ -1,8 +1,3 @@
 [
-    "flamenotes/flamenotes",
-    "larajournal/larajournal",
-    "gomzyakov/gomzyakov.github.io",
-    "github-profile-achievements/english",
-    "github-profile-achievements/russian",
-    "github-profile-achievements/github-profile-achievements.github.io"
+    "https://github.com/Idorenyin9010/good-first-issues.github.io/tree/good / Idorenyin"
```

Comment:
@Idorenyin9010 Hello!

Here you need to add a link to your own repository with issues with tag `good first issue`. Not a link to a fork of the current repository ;)